src/compositions/UserPopup/UserPopup.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react';
+
+import { AppRoutes } from '../../routes';
+import IconButton from '../IconButton/IconButton';
+
+import './UserPopup.scss';
+
+interface UserPopupProps {
+  address: string;
+  ENS?: string;
+  onLogoutClick: () => void;
+}
```

Comment:
```suggestion
  className?: string;
}
```

https://github.com/airswap/airswap-marketplace/blob/main/CONTRIBUTING.md#naming-conventions-and-order-of-properties